### ISSUE TITLE
Long Login and Sync times

### ISSUE THREAD

**Issue by losh11** [hivewallet/hive-js#246]
This was on the hive-us.heruko service, still extremely slow. It's like it's intentionally being forced to wait a few second before loading to the next screen. Why does it take so long in the first place?

It's takes from 30 seconds to one minute to login and sync, and even longer to create a new wallet. These times should be under 10 seconds, preferably 5 seconds.

The reason for all this is because people are lazy, and these days decide not to use apps which seem to load for long periods of time, and just stop using the app.

**Comment by ghost** [comment]
Yeah all of that could definitely be improved. But if you're (at all) referring to the initial load time, bear in mind how Heroku works:

https://stackoverflow.com/questions/2606190/why-are-my-basic-heroku-apps-taking-two-seconds-to-load

**Comment by losh11** [comment]
Can't Hive-JS be deployed on an AWS instance? Or even something like
DigitalOcean.

How much RAM, SSD space and compute power do you need? These can be easily
afforded. If it is a small instance, I'll pay for it.

I'm going to try to run this on a local instance and run diagnostics to see
why this is running slowly. If it's just Heroku being slow, I could help
contribute to running costs, maybe a crowdfunding service could help.
